### DISCUSSION TITLE
ARM support

### DISCUSSION THREAD

**Discussion by roifgroup** [papermerge/papermerge-core#290] [General]
First of all, thank you very much! Awesome work and the project is amazing.
I was wondering if there will be an ARM support?

I was trying to build it for ARM and got issues with the xapianpy package.
Working on rebuilding that.
https://github.com/roifgroup/xapian-bindings
Package output:
https://github.com/roifgroup/xapian-bindings/actions/runs/7358648229

Was wondering if there is a point in going further with this of not to bother with ARM support?

**Comment by ciur** [comment]
Frontend assets building (in github CI) take lot of time on ARM: less then 1 min on x86 vs 25-30 min on ARM.
Another reason why ARM is not there is that I simply have no device on which to test ARM based images.
